docs/mobile_sdk.md

```diff
@@ -3,14 +3,14 @@
 
 OpenWhisk provides a mobile SDK for iOS and watchOS 2 devices that enables mobile apps to easily fire remote triggers and invoke remote actions. A version for Android is currently not available; Android developers can use the OpenWhisk REST API directly.
 
-The mobile SDK is written in Swift 2.0 and supports iOS 9 and later releases.
+The mobile SDK is written in Swift 2.2 and supports iOS 9 and later releases.
 
 ## Adding the SDK to your app
-You can install the mobile SDK by using CocoaPods, or from the source directory.
+You can install the mobile SDK by using CocoaPods, Carthage, or from the source directory.
 
 ### Installing by using CocoaPods 
 
-The OpenWhisk SDK for mobile is available for public distribution through CocoaPods. The following lines in a Podfile install the SDK for an iOS app with a watchOS 2 extension:
+The OpenWhisk SDK for mobile is available for public distribution through CocoaPods. Assuming Cocoapods is installed, put the following lines into a file called 'Podfile' inside the starer app project directory. 
```

Comment:
The correct capitalization seems to be "CocoaPods" (see [cocoapods.org](https://cocoapods.org) for reference)